Design Patterns/Abstract Factory Pattern/src/GPUs/AmdMsiGPU.java

```diff
@@ -0,0 +1,10 @@
+package GPUs;
+
+public class AmdMsiGPU implements GpuBrand {
```

Comment:
there is no need to create a subclass without new fields, different logic, different implementation. so please add a fields, function..etc. to inner class.

Reply:
the different logic/implementation might be only in the assemble() method implementation but this example is showing a print statement for simplicity. 

I resolved this by mistake but it's not fully resolved yet, I will try to resolve it asap.